lib/tokenizers.js

```diff
@@ -256,6 +289,15 @@ async function _createTokenizer() {
       type: key.type
     }
   };
+  const collection = database.collections['tokenizer-tokenizer'];
+
+  if(explain) {
+    // 'find().limit(1)' is used here since 'find()' returns a cursor method
+    // which allows the use of the explain function.
+    const cursor = await collection.find(query).limit(1);
```

Comment:
Why are we using a `find` instead of an `update`?

Reply:
I can update the comment similar to your suggestion above. Update doesn't return a cursor method.

Reply:
Correct, but explain is not exclusive to cursors. Executing explain on collections have been supported since Mongo 3.0. Have you tried ` db.collection.explain('executionStats').update`?
 
See: https://docs.mongodb.com/upcoming/reference/method/db.collection.explain/

Reply:
I did try this, however I was unsuccessful on getting it to work. Maybe bedrock-mongodb does not support this? I keep getting the error: `collection.explain is not a function`.

Reply:
Okay, that's cool with me. Though it'd be better to test with the actual method, time spent debugging why it doesn't work isn't worth anybody's time at the moment.